graph/ops_convolve.go

```diff
@@ -43,30 +24,16 @@ type ConvolutionBuilder struct {
 	axes               ConvolveAxesConfig
 }
 
-// Convolve prepares a convolution on x with the given kernel for arbitrary
-// number of spatial dimensions (1D, 2D, 3D, etc.).
-//
-// It returns a ConvolutionBuilder object that can be further configured. Once the
-// configuration is finished, call `ConvolutionBuilder.Done` and it will return
-// the convolved x. Browse through ConvolutionBuilder to see its capabilities
-// and defaults.
-//
-// The shapes of x should be `[batch, <spatial_dimensions...>, input_channels]` if
-// configured with `ConvolutionBuilder.ChannelsAxis(timage.ChannelsLast)`, the default.
-// If one sets `ConvolutionBuilder.ChannelsAxis(timage.ChannelsFirst)`, the shapes should be
-// `[batch, input_channels, <spatial_dimensions...>]` instead.
-//
-// Note: `timage` refers to package `github.com/gomlx/gomlx/types/tensor/image`.
-//
-// The shapes of kernel should be `[<spatial_dimensions...>, input_channels, output_channels]` if
-// configured with `ConvolutionBuilder.ChannelsAxis(timage.ChannelsLast)`, the default. If one
-// sets `ConvolutionBuilder.ChannelsAxis(timage.ChannelsFirst)`, the shapes should be
-// `[input_channels, <spatial_dimensions...>, output_channels]` instead.
-//
-// Notice x and kernel must have the same rank.
-//
-// We follow the Keras convention of calling the "depth" or "feature" or "channels" dimension
-// "channels". Likewise, we use "kernel" instead of "filters" -- but they mean the same.
+func (conv *ConvolutionBuilder) FeatureGroupCount(groupCount int) *ConvolutionBuilder {
```

Comment:
I was going to suggest to follow the convention of having the methods below the function that creates the object (`Convolve` in this case), below the other similar methods that set optional parameters ?

Also, for the users, comments on what these parameters do. Maybe with a link to the original explanation in TensorFlow?).